korman/operators/op_toolbox.py

```diff
@@ -174,33 +174,33 @@ def execute(self, context):
 
         
 class PlasmaToggleDoubleSidedOperator(ToolboxOperator, bpy.types.Operator):
-    bl_idname = "mesh.plasma_toggle_double_sided"
+    bl_idname = "mat.plasma_toggle_double_sided"
     bl_label = "Toggle All Double Sided"
-    bl_description = "Toggles all meshes to be double sided"
+    bl_description = "Toggles all materials to be double sided"
     
     enable = BoolProperty(name="Enable", description="Enable Double Sided")
     
     def execute(self, context):
         enable = self.enable
-        for mesh in bpy.data.meshes:
-            mesh.show_double_sided = enable
+        for mat in bpy.data.materials:
+            mat.plasma_mat.plasma_double_sided = enable
         return {"FINISHED"}
 
 
 class PlasmaToggleDoubleSidedSelectOperator(ToolboxOperator, bpy.types.Operator):
-    bl_idname = "mesh.plasma_toggle_double_sided_selected"
+    bl_idname = "mat.plasma_toggle_double_sided_selected"
     bl_label = "Toggle Selected Double Sided"
-    bl_description = "Toggles selected meshes double sided value"
+    bl_description = "Toggles selected meshes' material(s) double sided value"
     
     @classmethod
     def poll(cls, context):
         return super().poll(context) and hasattr(bpy.context, "selected_objects")
 
     def execute(self, context):
-        mesh_list = [i.data for i in context.selected_objects if i.type == "MESH"]
-        enable = not all((mesh.show_double_sided for mesh in mesh_list))
-        for mesh in mesh_list:
-            mesh.show_double_sided = enable
+        mat_list = [i.data for i in context.selected_objects if i.type == "MATERIAL"]
+        enable = not all((mat.plasma_mat.plasma_double_sided for mat in mat_list))
+        for mat in mat_list:
+            mat.plasma_mat.plasma_double_sided = enable
```

Comment:
Unfortunately, `MATERIAL` isn't a valid object type. It might be easier for us to chain the material slots, so something like...

```python
        mat_list = [slot.material for slot in itertools.chain.from_iterable(i.material_slots for i in context.selected_objects) if slot and slot.material]
```

(Warning: untested)